src/app/games/(not-in-game)/results/[gameId]/page.tsx

```diff
@@ -35,6 +37,14 @@ async function getResults(gameId: string) {
 
   const { players, game } = data
 
+  const question = await db.query.questions.findFirst({
+    where: cmp.eq(schema.questions.id, game.question_id),
+  })
+
+  if (!question) {
```

Comment:
we should handle in the ui if question doesn't exist / is `null`, e.g. if we every end up deleting a question